spec/rubocop/cop/style/min_max_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+describe RuboCop::Cop::Style::MinMax, :config do
+  subject(:cop) { described_class.new(config) }
+
+  context 'with an array literal containing calls to `#min` and `#max`' do
+    it 'registers an offense if the receivers match' do
+      expect_offense(<<-RUBY.strip_indent)
```

Comment:
I guess you should add one example of `return foo.min, foo.max`.

Reply:
Ah. Yes. Perhaps parallel assignment, too. `bar = foo.min, foo.max`